utils/src/TrackSmearingTool.cxx

```diff
@@ -18,8 +18,8 @@ TrackSmearingTool::TrackSmearingTool(const std::string& smearingfile,
         throw std::invalid_argument( "Provided input smearing file does not exists");
 
     //cache the smearing histograms
-    smearing_histo_top_ = (TH1D*) smearingfile_->Get((tracks+"_p_vs_nHits_hh_smearing"+hsuffix).c_str());
-    smearing_histo_bot_ = (TH1D*) smearingfile_->Get((tracks+"_p_vs_nHits_hh_smearing"+hsuffix).c_str());
+    smearing_histo_top_ = (TH1D*) smearingfile_->Get((tracks+"_p_vs_nHits_hh_smearing"+hsuffix+"_top").c_str());
+    smearing_histo_bot_ = (TH1D*) smearingfile_->Get((tracks+"_p_vs_nHits_hh_smearing"+hsuffix+"_bot").c_str());
```

Comment:
This makes the old smearing file unusable right? I think we should either remove the old one or have the smearing tool support a file like the old one where the top and bottom are not separate.

Reply:
yeah good point, I don't think there's a reason to keep the old file around